.circleci/common.sh

```diff
@@ -109,6 +109,11 @@ function run_torch_xla_tests() {
       # python test/test_train_mp_mnist_amp.py --fake_data
     # fi
 
+    echo "Running Syncfree Optimizer Test"
+    if [ -x "$(command -v nvidia-smi)" ]; then
+      python test/test_syncfree_optimizers.py
```

Comment:
nit, can you remove 
```
    # if [ -x "$(command -v nvidia-smi)" ]; then
      # python test/test_train_mp_mnist_amp.py --fake_data
    # fi
```
above and add a 
```
# python test/test_train_mp_mnist_amp.py --fake_data
```
after your test. This way we group all gpu tests together(although it is commented out).